cmd/gazelle/update-repos.go

```diff
@@ -304,7 +304,10 @@ func updateRepos(wd string, args []string) (err error) {
 
 	updatedFiles := make(map[string]*rule.File)
 	for _, f := range sortedFiles {
-		merger.MergeFile(f, emptyForFiles[f], genForFiles[f], merger.PreResolve, kinds)
+		// We don't need to pass any wrapper macros config into MergeFile because the update repos command does not support
+		// the '# gazelle:alias_kind MACRO KIND' directive.
+		emptyWrapperMacros := map[string]string{}
+		merger.MergeFile(f, emptyForFiles[f], genForFiles[f], merger.PreResolve, kinds, emptyWrapperMacros)
```

Comment:
NIt: can you pass in `nil` instead of allocating a `map[string]string{}`? Read operations on a `nil` map work the same as an empty map, it's just the write operations that panic, and I don't think `MergeFile` needs to write this map.